src/systems/audio-debug-system.js

```diff
@@ -154,14 +156,16 @@ AFRAME.registerSystem("audio-debug", {
     this.zones.forEach(zone => {
       zone.el.setAttribute("audio-zone", "debuggable", enabled);
     });
-    const envRoot = document.getElementById("environment-root");
-    const meshEl = envRoot.querySelector(".trimesh") || envRoot.querySelector(".navMesh");
-    if (meshEl) {
+
+    const collisionEntities = [].slice.call(this.el.sceneEl.querySelectorAll("[nav-mesh]"));
```

Comment:
Would be more idiomatic to use `Array.from()` here

Reply:
Updated